README.md

```diff
@@ -89,7 +89,7 @@ public class MainApplication extends Application implements ReactApplication {
 
 1. Run `npm install react-native-mail --save`
 2. Open your project in XCode, right click on `Libraries` and click `Add
-   Files to "Your Project Name"` [(Screenshot)](http://url.brentvatne.ca/jQp8) then [(Screenshot)](http://url.brentvatne.ca/1gqUD).
+   Files to "Your Project Name"` [(Screenshot)](http://url.brentvatne.ca/jQp8) then [(Screenshot)](https://github.com/pedramsaleh/react-native-mail/blob/master/add-xcodeproj.png?raw=true).
```

Comment:
Yes that's what I'm saying :). If the file is saved into the rnmail repo, it could just be linked there and I could remove the reference to my repo. Unfortunately I don't have access to upload a file, so I can't do it.